locales/locales.php

```diff
@@ -67,7 +67,7 @@ public function __get( $name ) {
 
 	public function combined_name() {
 		/* translators: combined name for locales: 1: name in English, 2: native name */
-		return sprintf( _x( '%1$s/%2$s', 'locales' ), $this->english_name, $this->native_name );
+		return sprintf( _x( '%1$s/%2$s', 'Locales', 'glotpress' ), $this->english_name, $this->native_name );
```

Comment:
```suggestion
		return sprintf( __( '%1$s/%2$s', 'glotpress' ), $this->english_name, $this->native_name );
```

Doesn't seem like we need a context here since the string is only used once.

Reply:
Done.